sig-docs/annual-report-2021.md

```diff
@@ -0,0 +1,125 @@
+
+<!-- TODO -->
+
+## Membership
+
+**Are all listed SIG leaders (chairs, tech leads, and subproject owners) active?**
+
+Yes, they are.
+
+**How do you measure membership? By mailing list members, OWNERs, or something else?**
+
+We don't have an official way of measuring membership.
+
+**How does the group measure reviewer and approver bandwidth? Do you need help in any area now? What are you doing about it?**
+
+PR are generally open for anyone to review. We have [PR wrangler rotation](https://github.com/kubernetes/website/wiki/PR-Wranglers).
+In general each of the approver will taking the rotation. Each approver participates in the PR wrangler rotation. Approvers are supported during their rotation by other reviewers and approvers.
+
+For technical reviews, other than SIG Docs Tech Leads, we also notify respective SIGs with necessary knowledge of the technical aspect of the content. SIG Docs receives contributions from community members representing different SIGs. During the PR review process, SIG Docs reaches out to other SIGs when a pull request requires technical review.
+
+**Is there a healthy onboarding and growth path for contributors in your SIG? 
+What are some activities that the group does to encourage this? 
+What programs are you participating in to grow contributors throughout the contributor ladder?**
+
+There is no official onboarding process. However, the SIG is supportive of new contributors,
+willing to walk through the contribution process and eager to see contributions completed.
+Issues are created regularly and triaged. Issues are also created specifically for new 
+contributors. There are issues labelled for the different contribution areas (web development, 
+blog, first time contributors, task, concept, API, etc) which helps new contributors 
+identify how to get started.
+
+From time to time we use the weekly meeting Zoom call to help new contributors get onboard.
+
+**What programs do you participate in for new contributors?**
+
+We have an in person session during the KubeCon.
+
+**Does the group have contributors from multiple companies/affiliations? 
+
+Yes
+
+Can end users/companies contribute in some way that they currently are not?**
+
+I suppose we're particular keen to hear about suggested improvements from people who are using Kubernetes in production and can see gaps in what's there.
+
+## Current initiatives and project health
+
+[ ] Please include links to KEPs and other supporting information that will be beneficial to multiple types of community members.
+
+Longer term projects include
+
+- co-ordination with WG naming for things like removing the word “slave” (and other problematic terms) from docs.
+- publishing better information about releases kubernetes/website#20293
+
+**What are initiatives that should be highlighted, lauded, shout out, that your group is proud of? 
+Currently underway? What are some of the longer tail projects that your group is working on?**
+
+<!-- TODO -->
```

Comment:
It would be good to call out the docsy theme work as well as the reference documentation generation work.